src/application.js

```diff
@@ -0,0 +1,24 @@
+export default class ApplicationView {
+  constructor(element) {
+    this.element = element;
+    this.data = [];
+  }
+  render() {
+    const list = this.element.querySelector('.puppy-list');
+    list.innerHTML = '';
+    debugger;
+
+    // this.data.forEach((puppy) => {
```

Comment:
Uncomment this and then make sure to import `PuppyView` since it is declared in a different module.

Then try running `newPuppy.render()` similar to the way we rendered list items in class: https://github.com/tiy-tn-fee-2016-summer/lesson-week-05/blob/day-04/src/question-list.js#L55